aa_stripe/api.py

```diff
@@ -30,6 +32,18 @@ def get_serializer_class(self):
         return self.serializer_class
 
 
+class StripeCardsDetailsAPI(RetrieveUpdateDestroyAPIView):
+    queryset = StripeCard.objects.all()
+    serializer_class = StripeCardListSerializer
+    permission_classes = (IsAuthenticated,)
```

Comment:
Security issue. This way I can see update, delete any card data. This should show only logged in user cards.